acquia_lift.module

```diff
@@ -39,3 +38,38 @@ function acquia_lift_page_attachments(array &$page) {
   // Attach lift CSS to all pages.
   $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function acquia_lift_toolbar() {
+  $items = [];
+
+  $items['acquia_lift'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'html_tag',
+      '#tag' => 'button',
+      '#value' => t('Acquia Lift'),
+      '#attributes' => [
+        'title' => t('Acquia Lift'),
+        'class' => ['toolbar-icon', 'toolbar-icon-acquia-lift'],
+        // This particular id will make sure the JS behavior is attached to the
+        // link.
+        'id' => ['openLiftLink'],
+        'aria-pressed' => 'false',
+      ],
+    ],
+    '#access' => AccessResult::allowedIfHasPermission(\Drupal::currentUser(), 'access acquia lift links'),
+    '#wrapper_attributes' => [
+      'class' => ['acquia-lift-toolbar-tab'],
+    ],
+    '#attached' => [
+      'library' => [
+        'acquia_lift/acquia-lift',
+      ],
+    ],
+  ];
+
+  return $items;
+}
```

Comment:
The following would be how I would have approached this:

* Create Drupal\acquia_lift\Service\Helper\ShortcutHelper class, just like the HelpMessageHelper class.
* Move all logic into there.
* Unit test the Helper.
* Simpletest / functionally test the button exists.

This is to follow MVC's "fat-model, skinny-controller" guideline.

This note is FYI only, I wouldn't demand other developers to do the same - I'll be okay if you skip this note in partial or full.